sources/packages/backend/workflow/src/workflow-definitions/parttime-assessment-2022-2023.bpmn

```diff
@@ -1120,8 +1120,9 @@ dmnX (comes from dmn)</bpmn:text>
           <zeebe:output source="=calculatedDataStudentFinancialInformation.calculatedDataCurrentYearIncome" target="calculatedDataCurrentYearIncome" />
           <zeebe:output source="=calculatedDataStudentFinancialInformation.calculatedDataDaycareCosts12YearsOrOver" target="calculatedDataDaycareCosts12YearsOrOver" />
           <zeebe:output source="=calculatedDataStudentFinancialInformation.calculatedDataDaycareCosts11YearsOrUnder" target="calculatedDataDaycareCosts11YearsOrUnder" />
+          <zeebe:output source="=calculatedDataStudentFinancialInformation.calculatedDataCurrentYearPartnerIncome" target="calculatedDataCurrentYearPartnerIncome" />
         </zeebe:ioMapping>
-        <zeebe:script expression="={&#10; calculatedDataTaxReturnIncome:  studentDataTaxReturnIncome,&#10; calculatedDataCurrentYearIncome: (if studentDataCurrentYearIncome = null then null else studentDataCurrentYearIncome),&#10; calculatedDataDaycareCosts12YearsOrOver: (if studentDataDaycareCosts12YearsOrOver = null then 0 else studentDataDaycareCosts12YearsOrOver),&#10; calculatedDataDaycareCosts11YearsOrUnder: (if studentDataDaycareCosts11YearsOrUnder = null then 0 else studentDataDaycareCosts11YearsOrUnder)&#10;}" resultVariable="calculatedDataStudentFinancialInformation" />
+        <zeebe:script expression="={&#10; calculatedDataTaxReturnIncome:  studentDataTaxReturnIncome,&#10; calculatedDataCurrentYearIncome: (if studentDataCurrentYearIncome = null then null else studentDataCurrentYearIncome),&#10; calculatedDataDaycareCosts12YearsOrOver: (if studentDataDaycareCosts12YearsOrOver = null then 0 else studentDataDaycareCosts12YearsOrOver),&#10; calculatedDataDaycareCosts11YearsOrUnder: (if studentDataDaycareCosts11YearsOrUnder = null then 0 else studentDataDaycareCosts11YearsOrUnder),&#10;  calculatedDataCurrentYearPartnerIncome: (if studentDataCurrentYearPartnerIncome = null then null else studentDataCurrentYearPartnerIncome)&#10;}" resultVariable="calculatedDataStudentFinancialInformation" />
```

Comment:
The current year partner is not available in student application form.  Please hard code it to null in the student path. 

`studentDataCurrentYearPartnerIncome: null`